__test__/unitTest/removeInvalidTokensFromList.test.ts

```diff
@@ -0,0 +1,33 @@
+// import { removeInvalidTokensFromList, validateTokenListWithErrorThrowing, tokenListIsValid} from '../../src/lib/utils'
+import { removeInvalidTokensFromList } from "../../src/lib/utils"
+import arblist from "./schma/arbify.tokenlist.json";
+import arblistdecimalsTooLow from "./schma/arbifyInvalid1.tokenlist.json"
+import arblistWrongVersion from "./schma/arbifyInvalid7.tokenlist.json"
```

Comment:
We might want to rename the file, so it's clear what are the issues with each list. `decimalsTooLow.json`, `invalidVersion.json`, etc...